PowerUp-2018/src/com/spikes2212/robot/Robot.java

```diff
@@ -31,16 +32,16 @@
 	public static BasicSubsystem liftLocker;
 	public static BasicSubsystem lift;
 	public static TankDrivetrain drivetrain;
-
+	public static DashBoardController dbc = new DashBoardController();
 
 	/**
-	 * This function is run when the robot is first started up and should be used
-	 * for any initialization code.
+	 * This function is run when the robot is first started up and should be
+	 * used for any initialization code.
 	 */
 	@Override
 	public void robotInit() {
 		roller = new BasicSubsystem(SubsystemComponents.Roller.MOTOR::set,
-				new TwoLimits(() -> true, SubsystemComponents.Roller.LIGHT_SENSOR::get));
+				new TwoLimits(() -> false, () -> !SubsystemComponents.Roller.LIGHT_SENSOR.get()));
 		drivetrain = new TankDrivetrain(SubsystemComponents.Drivetrain.LEFT_MOTOR::set,
```

Comment:
you have used LIGHT_SENSOR.get() instead of LIGHT_SENSOR::get

Reply:
I used .get() so i would be able to reverse it